pkg/containerspec/containerspec.go

```diff
@@ -42,16 +42,18 @@ func (n KubeResourceQuantity) MarshalJSON() ([]byte, error) {
 	return json.Marshal(string(n))
 }
 
-func (n KubeResourceQuantity) copy() (KubeResourceQuantity) {
+func (n KubeResourceQuantity) copy() KubeResourceQuantity {
 	return KubeResourceQuantity(string(n))
 }
 
 // PaastaContainerSpec : Spec for any paasta container with basic fields and utilities
 type PaastaContainerSpec struct {
-	CPU       *KubeResourceQuantity `json:"cpus"`
-	Memory    *KubeResourceQuantity `json:"mem"`
-	Disk      *KubeResourceQuantity `json:"disk"`
-	DiskLimit *KubeResourceQuantity `json:"disk_limit,omitempty"`
+	CPU         *KubeResourceQuantity `json:"cpus"`
```

Comment:
nit, but I think both cpus and cpus_limit should be singular to match the k8s api. I know that's a pain to change, so don't block on it

Reply:
Right, I can imagine rolling out such change would be right pain in the backside ... sorry, not going to do it 😞 (for this PR, at least)